modules/021-kube-proxy/images/init-container/Dockerfile

```diff
@@ -3,7 +3,7 @@ ARG BASE_GOLANG_19_ALPINE
 FROM $BASE_GOLANG_19_ALPINE as artifact
 
 WORKDIR /src/
-COPY main.go go.mod go.sum /src/
+COPY init/ /src/
```

Comment:
In cni-flannel we have `entrypoint` folder, here name is `init`. Can we stick to one naming?

Reply:
Fixed.